service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -105,6 +172,95 @@ public void insertEvidenceSummaryDocument(GeneratePdfPayload request, String doc
     claimRepository.save(claim);
   }
 
+  @Override
+  @Transactional
+  public void insertOrUpdateExamOrderingStatus(ExamOrder examOrder) {
+    ExamOrderEntity examOrderEntity =
+        examOrderRepository
+            .findByCollectionId(examOrder.getCollectionId())
+            .orElseGet(() -> createExamOrder(examOrder));
+    if (null != examOrderEntity) {
+      examOrderEntity.setCollectionId(examOrder.getCollectionId());
+      examOrderEntity.setStatus(examOrder.getStatus());
```

Comment:
In subsequent PR, remember to update `status_at` with the payload's `examOrderDateTime`.